.github/workflows/ci-advanced.yml

```
@@ -0,0 +1,352 @@
+        uses: docker/setup-buildx-action@v3
+
+      - name: Run Molecule tests
+        run: |
+          cd molecule/${{ matrix.scenario }}
+          molecule test
+        env:
+          PY_COLORS: '1'
+          ANSIBLE_FORCE_COLOR: '1'
+
+      - name: Upload test results
+        uses: actions/upload-artifact@v3
```

actionlint (1.7.7)

the runner of "actions/upload-artifact@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)